File: clean.py
import re

file_path = 'C:/Users/Nexrom/Desktop/PF/PortfolioNexrom/index.html'
with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Using regex to remove the CashCash article block
content = re.sub(
    r'\s*<!-- Project 3: CashCash -->.*?<h3 class="cell-title">CashCash</h3>.*?</article>', 
    '', 
    content, 
    flags=re.DOTALL
)

# Using regex to remove the CashCash table row
content = re.sub(
    r'\s*<tr>\s*<td>\s*<span class="project-title">CashCash \(AP 2 - Client Lourd\)</span>.*?\s*</tr>',
    '',
    content,
    flags=re.DOTALL
)

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)
print("Done")

File: clean.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'index.html');
let content = fs.readFileSync(filePath, 'utf8');

// Regex to remove the CashCash bento cell
content = content.replace(/\s*<!-- Project 3: CashCash -->[\s\S]*?<h3 class="cell-title">CashCash<\/h3>[\s\S]*?<\/article>/g, '');

// Regex to remove the CashCash row in the skills table
content = content.replace(/\s*<tr>\s*<td>\s*<span class="project-title">CashCash \(AP 2 - Client Lourd\)<\/span>[\s\S]*?<\/tr>/g, '');

fs.writeFileSync(filePath, content, 'utf8');
console.log('done');
